packages/admin/admin/src/section/SectionHeadline.tsx

```diff
@@ -0,0 +1,138 @@
+
+export type SectionHeadlineClassKey = "root" | "headlineWrapper" | "titleContainer" | "divider" | "supportText" | "infoTooltipText" | "infoIcon";
+
+const Root = createComponentSlot("div")<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "root",
+})();
+
+const HeadlineWrapper = createComponentSlot("div")<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "headlineWrapper",
+})(
+    () => css`
+        display: flex;
+        flex-direction: row;
+        align-items: flex-end;
+        justify-content: space-between;
+    `,
+);
+
+const TitleContainer = createComponentSlot("div")<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "titleContainer",
+})(
+    () => css`
+        display: flex;
+        align-items: center;
+    `,
+);
+
+const SupportText = createComponentSlot(Typography)<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "supportText",
+})(
+    ({ theme }) => css`
+        color: ${theme.palette.text.secondary};
+    `,
+);
+
+const StyledTooltip = createComponentSlot(Tooltip)<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "infoTooltipText",
+})(
+    ({ theme }) => css`
+        color: ${theme.palette.text.secondary};
+        margin-left: 10px;
+    `,
+);
+
+const StyledDivider = createComponentSlot(Divider)<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "divider",
+})(
+    () => css`
+        margin-top: 10px;
+        color: "inherit";
+    `,
+);
+
+const StyledInfoIcon = createComponentSlot(Info)<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "infoIcon",
+})(
+    () => css`
+        font-size: 12px;
+    `,
+);
+
+export interface SectionHeadlineProps
+    extends ThemedComponentBaseProps<{
+        root: "div";
+        headlineWrapper: "div";
+        titleContainer: "div";
+        infoTooltipText: typeof Tooltip;
+        infoIcon: typeof Info;
+        supportText: typeof Typography;
+        divider: typeof Divider;
+    }> {
+    children: ReactNode;
+    divider?: boolean;
+    supportText?: string;
+    infoTooltipText?: string;
+}
+
+export function SectionHeadline(inProps: SectionHeadlineProps) {
+    const { children, divider, supportText, infoTooltipText, slotProps, ...restProps } = useThemeProps({
+        props: inProps,
+        name: "CometAdminSectionHeadline",
+    });
+
+    return (
+        <Root {...slotProps?.root} {...restProps}>
+            <HeadlineWrapper {...slotProps?.headlineWrapper}>
+                <TitleContainer {...slotProps?.titleContainer}>
+                    {children}
+                    {infoTooltipText && (
+                        <StyledTooltip title={infoTooltipText} {...slotProps?.infoTooltipText}>
+                            <StyledInfoIcon {...slotProps?.infoIcon} />
```

Comment:
Icons should not be slots, they should be defined as an overridable map, so the icon can be easily swapped out, if necessary, see the example in the Docs: https://docs.comet-dxp.com/docs/comet-core-development/creating-customizable-admin-components#overridable-icons

Reply:
Sorry, just saw your comment now, after reviewing again 😅 

> Is it okay to use inline styling for the icon in this case (as I need to set the fontSize to 12px?

See https://github.com/vivid-planet/comet/pull/4399/files#r2310004214

> I am also not 100% sure about the check, I used here: https://github.com/vivid-planet/comet/pull/4399/files#diff-9b77715803d1a58c3c27d7bd81d2a1be30d81194387f7da431c564c978b3ecd0R112 - I added it, as I get a type error otherwise.

See https://github.com/vivid-planet/comet/pull/4399/files#r2310015864